packages/nodes-base/nodes/SyncroMSP/SyncroMspTrigger.node.ts

```diff
@@ -0,0 +1,139 @@
+import {
+	IWebhookFunctions,
+} from 'n8n-core';
+
+import {
+	IDataObject,
+	INodeType,
+	INodeTypeDescription,
+	IWebhookResponseData,
+} from 'n8n-workflow';
+
+
+export class SyncroMspTrigger implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'SyncroMSP Trigger',
+		name: 'syncroMspTrigger',
+		icon: 'file:syncromsp.png',
+		group: ['trigger'],
+		version: 1,
+		subtitle: '=Events: {{$parameter["events"].includes("*") ? "All" :  $parameter["events"].join(", ")}}',
+		description: 'Starts the workflow when events occur on SyncroMSP',
+		defaults: {
+			name: 'SyncroMSP Trigger',
+			color: '#08a4ab',
+		},
+		inputs: [],
+		outputs: ['main'],
+		webhooks: [
+			{
+				name: 'default',
+				httpMethod: 'POST',
+				responseMode: 'onReceived',
+				path: 'webhook',
+			},
+		],
+		properties: [
+			{
+				displayName: 'To use this node, add the webhook URLs above in Synchro. <a href="https://help.syncromsp.com/hc/en-us/articles/115002529773-Notification-Center-Overview" target="_blank">More details</a>',
+				name: 'webhookNotice',
+				type: 'notice',
+				default: '',
+			},
+			{
+				displayName: 'Events',
+				name: 'events',
+				type: 'multiOptions',
+				options: [
+					{
+						name: '*',
```

Comment:
I talked with David and he suggested we use `All` instead of `*` for all events.

Considering it hasn't been released, I'd say we should update both the label and the value. Could you please apply this change? Everything else looks good.